libmikmod/loaders/load_amf.c

```diff
@@ -126,7 +126,16 @@ static BOOL AMF_UnpackTrack(MREADER* r)
 	/* read packed track */
 	if (r) {
 		tracksize=_mm_read_I_UWORD(r);
-		tracksize+=((ULONG)_mm_read_UBYTE(r))<<16;
+
+		/* The original code in DSMI library read the byte, but it is not used.
+		   Normally, it is zero, but Avoid.amf (version 8) have a track where its not.
+		   This track is garbage, so check the value and skip the track. */
```

Comment:
It doesn't sound garbage to me in OpenMPT (after ignoring this byte). What makes you think it's garbage?